src/hooks/wallets/wallets.ts

```diff
@@ -76,3 +76,7 @@ export const getSupportedWallets = (disabledWallets: string[]): WalletInit[] =>
 export const isHardwareWallet = (wallet: ConnectedWallet): boolean => {
   return [WALLET_KEYS.LEDGER, WALLET_KEYS.TREZOR].includes(wallet.label.toUpperCase() as WALLET_KEYS)
 }
+
+export const isPaired = (wallet: ConnectedWallet): boolean => {
```

Comment:
Not a good name. Should be similar to `isHardwareWallet` instead. E.g. `isSafeMobileWallet`.

Reply:
Fixed [here](https://github.com/safe-global/web-core/pull/537).